cypress/component/InvalidObject.cy.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import InvalidObject from '../../packages/module/dist/dynamic/InvalidObject';
+
+describe('InvalidObject', () => {
+  /* eslint-disable no-console */
```

Comment:
Is this one needed?

Reply:
oh :sweat_smile: the problem of copy-paste. I took these out.